torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -54,12 +54,12 @@ def __call__(
         self,
         op: torch._ops.OpOverload,
         mutated_args_names: List[str],
-        kwargs: Dict[str, Any],
+        **kwargs: Dict[str, Any],
```

Comment:
this is a bit weird in the case that the operator has a kwargs named "op" or "mutated_arg_names".

```
>>> def foo(a, **kwargs):
...     print(a, kwargs)
... 
>>> foo(10, a=5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() got multiple values for argument 'a'
```

At least we get a loud error in this case, so I'm okay with it in principle. Maybe rename `op` to something less common and prefix both arguments with `_` to reduce the chance of collision.

This comment applies to every function signature where we did this change.